xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -99,17 +131,71 @@ public static CertificateProviderInfo create(String pluginName, Map<String, ?> c
     @Nullable
     public abstract ImmutableMap<String, CertificateProviderInfo> certProviders();
 
+    /**
+     * A template for the name of the Listener resource to subscribe to for a gRPC server.
+     *
+     * <p>If starts with "xdstp:", will be interpreted as a new-style name, in which case the
```

Comment:
if using new style, e.g. `xdstp://foo.google.com` then it would look up server listener template in the authority map with the key `foo.google.com`? If so, it basically does nothing.

Reply:
oh no problem, i was not familiar with the design doc. 
I was just imagining the lookup at the server side will be at runtime, currently it looks hardcode in the bootstrap.